conda_smithy/feedstock_content/LICENSE

```diff
@@ -1,3 +1,7 @@
+A separate license may cover the recipe itself.  Please see the recipe
+directory for a LICENSE, recipe-license.txt or similar file for this
```

Comment:
Let's pick one of these for consistency. My vote is some sort of `recipe-license.txt` or `RECIPE_LICENSE.txt` or similar. As long as it starts with "recipe", has "license", and has ".txt" extension it is fine.